src/app/app.component.spec.ts

```diff
@@ -5,7 +5,10 @@ import { AppComponent } from './app.component';
 import { StoreModule } from '@ngrx/store';
 import { RouterTestingModule } from '@angular/router/testing';
 
-import { RouterOutletStubComponent } from '../testing';
+import { Component } from '@angular/core';
+
+@Component({selector: 'router-outlet', template: ''})
+export class RouterOutletStubComponent { }
```

Comment:
@sudheesh001 Yes This is a outlet stub component required for unit tests to properly run on AppComponent